src/vm/checker/read_only/tests.rs

```diff
@@ -26,6 +26,14 @@ fn test_simple_read_only_violations() {
          (define (func1) (set-entry! tokens (tuple (account tx-sender)) (tuple (balance 10))))
          (define-read-only (not-reading-only)
             (map + (list 1 (set-entry! tokens (tuple (account tx-sender)) (tuple (balance 10))) 3)))",
+        "(define-map tokens ((account principal)) ((balance int)))
+         (define (update-balance-and-get-tx-sender)
+            (begin              
+              (set-entry! tokens (tuple (account tx-sender))
+                                 (tuple (balance 10)))
+              tx-sender))
+         (define-read-only (get-balance)
+            (fetch-entry tokens (tuple (account (update-balance-and-get-tx-sender)))))",
```

Comment:
Can this test be changed to use the implicit tuple type, e.g.:

```
(define-read-only (get-balance)
            (fetch-entry tokens ((account (update-balance-and-get-tx-sender)))))
```

That way it'll test the `are_tuple_values_read_only` function.